core/src/OpenACC/Kokkos_OpenACC_ParallelReduce_Team.hpp

```diff
@@ -337,7 +337,6 @@ parallel_reduce(const Impl::TeamThreadRangeBoundariesStruct<
   value_type tmp;
   wrapped_reducer.init(&tmp);
 
-  ValueType tmp = ValueType();
```

Comment:
How did we not notice that we tried declaring the same variable twice?

Reply:
I guess this occurred while merging (NVHPC OpenACC compiler didn't complain this, while Clacc LLVM compiler does.)